Documentation/docs-site/content/usage/executor-types.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Executor Types"
+date: 2018-02-01T23:10:05-07:00
+draft: false
+weight: 41
+---
+## Executors and ExecutorType
+
+When you create a function, you can specify an executortype for a function. An executortype controls how function pods are created and what capabilities are available for that executor type. 
+
+Poolmgr is the default executortype for functions and has been part of Fission since inception. NewDeploy is a new executortype added in v0.5.0 and is in alpha state. Following sections detail each of executor types and what capabilities are available to the user.
+
+### Resource requirements
+
+From release 0.5.0 you can also specify resource requirements for function pods. The behaviour varies for both executortypes and is explained in respective section.
+
+### Pool Manager (poolmgr)
+
+A pool manager executortype (Refered to as poolmgr) creates a pool of generic environment pods as soon as you create an environment. The pool size of initial "warm" containers can be configured based on user needs. These warm containers contain a small dynamic loader.
+
+Resource requirements are specified at environment level and are inherited by specialized function pods. 
+
+Once you create a function and invoke it, one of pods from the pool is taken out and "specialized" and used for execution. This pod is used for subseqnent requests in a serialized manner. If there are no more requests for a certain idle duration, then this pod is cleaned up. If a new requests come after the earlier specialized pod was cleaned up, then a new pod is specialised from the pool and used for execution.
+
+Poolmgr executortype is great for functions where lower latency is a requirement. Poolmgr executortype has certain limitations for example you can not autoscale them based on demand.
+
+
+### New Deploy (newdeploy)
+
+NewDeploy executortype creates a Kubernetes Deployment along with a Service and HorizontalPodAutoscaler for function execution. This enables autoscaling of function pods and load balancing the requests between pods. In future additional capabilities can be added for newdeploy executortype such as support for volume etc. 
```

Comment:
~the~ requests between pods

in _the_ future, additional ...

...for the new-deploy executor, such as support for volumes, etc.